.gitignore

```diff
@@ -25,6 +25,7 @@ coverage-acceptance.txt
 __pycache__
 *.pyc
 
+.idea/
```

Comment:
Given that this is unrelated to the main topic of the PR, it should go into separate PR (we might want to revert one or the other change, bundling them together would complicate it).